server/lib/routes/get-index.js

```diff
@@ -9,9 +25,44 @@ module.exports = function () {
   route.path = '/';
 
   route.process = function (req, res) {
-    res.render('index');
+    if (! shouldEmitActivityEvent()) {
+      return res.render('index');
+    }
+
+    var time = Date.now();
+    var flowId = crypto.randomBytes(16).toString('hex');
+    res.render('index', { flowBeginTime: time, flowId: flowId });
+
+    setImmediate(function () {
+      var userAgent = ua.parse(req.headers['user-agent']);
+      // This event data is structured to be consistent with
+      // the auth server activity events.
+      var activityEvent = {
+        /*eslint-disable camelcase*/
+        device_id: null,
+        event: 'flow.begin',
+        flow_id: flowId,
+        flow_time: 0,
+        new_user: false,
+        time: time,
+        uid: null,
+        user_agent_browser: userAgent.ua.family,
+        user_agent_os: userAgent.os.family + ' ' + userAgent.os.major,
+        user_agent_version: userAgent.ua.major
+        /*eslint-enable camelcase*/
+      };
+      activityEventQueryParams.forEach(function (key) {
+        activityEvent[key] = req.query[key] || null;
+      });
+      // The data pipeline only reads from stderr.
+      process.stderr.write('activityEvent ' + JSON.stringify(activityEvent) + '\n');
```

Comment:
> ...maybe we should actually re-use the auth server's `Lug` object in the content server?

Hmmm, maybe not wholesale as there's stuff in there that we probably don't want shared. But we could definitely extract some of it, even if it was just the guts of `activityEvent`.